src/main/java/duke/Event.java

```diff
@@ -0,0 +1,28 @@
+package duke;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.time.format.FormatStyle;
+
+public class Event extends Task {
+    protected LocalDate at;
```

Comment:
Might want to consider changing the variable name to 'timing' instead because it wasn't very obvious what this variable was referring to at first glance.